build.gradle

```diff
@@ -0,0 +1,91 @@
+plugins {
+  id 'com.jfrog.bintray' version '1.8.4'
+  id 'java-gradle-plugin'
+  id 'net.researchgate.release' version '2.5.0'
+}
+
+apply plugin: 'com.jfrog.bintray'
+apply plugin: 'groovy'
+apply plugin: 'maven-publish'
+
+gradlePlugin {
+  plugins {
+    updateChangelogPlugin {
+      id = 'com.tomtom.gradle.updatechangelog'
+      displayName = 'Gradle Update Changelog Plugin'
+      description = 'A plugin that automatically updates the version in your changelog file.'
+      implementationClass = 'com.tomtom.gradle.updatechangelog.UpdateChangelogPlugin'
+    }
+  }
+}
+
+wrapper {
+  gradleVersion = '4.10'
```

Comment:
By calling gradle wrapper it basically creates all the required files (wrapper.properties, gradlew, gradlew.bat, etc.) based on the specified version.

This version just means that when calling `gradlew wrapper` it will download version 4.10. The version in wrapper.properties is the one specified here.

See more https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:adding_wrapper.